README.md

```diff
@@ -138,3 +138,34 @@ Changes are tracked as [GitHub releases](https://github.com/graphql/graphql-js/r
 ### License
 
 GraphQL.js is [MIT-licensed](https://github.com/graphql/graphql-js/blob/master/LICENSE).
+
+## TypeScript definition files
+
+This module includes a [TypeScript](https://www.typescriptlang.org/)
+declaration file to enable auto complete in compatible editors and type
+information for TypeScript projects.
+
+### Credits
+
+The `*.d.ts` files in this folder are based on [DefinetlyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/54712a7e28090c5b1253b746d1878003c954f3ff/types/graphql) definitions written by:
```

Comment:
Some different wording like this `We used `*.d.ts` files from [DefinetlyTyped]() as basis for TypeScript convertion in `v15.0.0`. Big thanks to:`
Or something similar